README.md

```diff
@@ -230,7 +230,11 @@ runc delete mycontainerid
 This allows higher level systems to augment the containers creation logic with setup of various settings after the container is created and/or before it is deleted. For example, the container's network stack is commonly set up after `create` but before `start`.
 
 #### Rootless containers
-`runc` has the ability to run containers without root privileges. This is called `rootless`. You need to pass some parameters to `runc` in order to run rootless containers. See below and compare with the previous version. Run the following commands as an ordinary user:
+`runc` has the ability to run containers without root privileges. This is called `rootless`. You need to pass some parameters to `runc` in order to run rootless containers. See below and compare with the previous version.
+
+**Note:** `userns` must be compiled and enabled in your kernel. To enable this feature, run `echo 1 > /proc/sys/kernel/unprivileged_userns_clone`.
```

Comment:
This is specific to Debian and Arch https://git.archlinux.org/linux.git/commit/?h=v4.19.1-arch1&id=16e003599a65153e8d64c2e5ae46ff6a03862daf

Reply:
Not to mention that you should probably also check `CONFIG_USER_NS=y` in `/proc/config.gz`.

Reply:
Thanks for the review. I'll update the PR to reflect that.

Reply:
Updated.